- Remove hardcoding debug and optimization flags
- Avoid picking up net/libnet
- Don't run fc-cache on install

Index: configure.ac
--- configure.ac.orig
+++ configure.ac
@@ -24,9 +24,6 @@ dnl This has to happen before any other AC_* calls, so
 
 dnl Set VICE's compiler flags (can be overridden by the user)
 VICE_CPPFLAGS="-I\$(top_srcdir)/src/systemheaderoverride"
-VICE_CFLAGS="-g -O3"
-VICE_CXXFLAGS="-g -O3"
-VICE_OBJCFLAGS="-g -O3"
 VICE_LDFLAGS=""
 
 
@@ -2030,8 +2027,6 @@ if test x"$is_unix_x11" = "xyes" -o x"$is_unix_macosx"
                                          LIBS="-lsocket $LIBS"],,)
     AC_CHECK_LIB(bsd, gethostbyname,[ NETPLAY_LIBS="-lbsd $NETPLAY_LIBS";
                                       LIBS="-lbsd $LIBS"],,)
-    AC_CHECK_LIB(net, gethostbyname,[ NETPLAY_LIBS="-lnet $NETPLAY_LIBS";
-                                      LIBS="-lnet $LIBS"],,)
     AC_CHECK_LIB(inet, gethostbyname,[ NETPLAY_LIBS="-linet $NETPLAY_LIBS";
                                        LIBS="-linet $LIBS"],,)
 
@@ -2615,12 +2610,7 @@ if test x"$enable_lame" = "xyes" ; then
   CFLAGS="$old_CFLAGS"
 fi
 
-AC_PATH_PROGS(FCCACHE, fc-cache, false)
-if test x"$FCCACHE" = "xfalse"; then
-  AM_CONDITIONAL(HAVE_FC_CACHE, false)
-else
-  AM_CONDITIONAL(HAVE_FC_CACHE, true)
-fi
+AM_CONDITIONAL(HAVE_FC_CACHE, false)
 
 dsound_header_no_lib="no"
 
